cloudpathlib/cloudpath.py

```diff
@@ -245,10 +245,8 @@ def __del__(self) -> None:
             self._handle.close()
 
         # ensure file removed from cache when cloudpath object deleted
-        if (
-            hasattr(self, "client")
-            and self.client.file_cache_mode == FileCacheMode.cloudpath_object
-        ):
+        client = getattr(self, "client", object)
+        if getattr(client, "file_cache_mode", None) == FileCacheMode.close_file:
```

Comment:
This should still be `cloudpath_object`, not `close_file`. See https://cloudpathlib.drivendata.org/stable/caching/#automatically

Reply:
Oops, that was my bad, bad copy-pasta 😞 Thanks for the catch!